db/external_sst_file_ingestion_job.cc

```diff
@@ -687,6 +821,26 @@ Status ExternalSstFileIngestionJob::AssignGlobalSeqnoForIngestedFile(
   return Status::OK();
 }
 
+IOStatus ExternalSstFileIngestionJob::GenerateChecksumForIngestedFile(
+    IngestedFileInfo* file_to_ingest) {
+  if (db_options_.file_checksum_gen_factory == nullptr ||
+      ingestion_options_.generate_and_verify_file_checksum == false ||
+      ingestion_options_.write_global_seqno == false) {
```

Comment:
Maybe add a comment here to provide a little context on `write_global_seqno` case. Something like "if write_global_seqno is false, it means we are going to use file checksum computed during Prepare()".

Reply:
@riversand963 Sure. Thanks for the review!